pages/index.js

```diff
@@ -36,7 +36,7 @@ export default function HomePage() {
       </Head>
       <Navbar />
       <Landing
-        heading="We equip students with technical superpowers"
+        heading="test"
```

Comment:
Can you change this back to: "We equip students with technical superpowers" from "test" LOL

Reply:
I changed it back in our zoom, its not showing up as test in any of the branches I have